src/services/io/podio/make_datamodel_glue.py

```diff
@@ -112,10 +114,16 @@ def AddCollections(datamodelName, collectionfiles):
     f.write('#pragma once\n')
     f.write('\n')
     f.write('#include <stdexcept>\n')
+    f.write('#include <podio/podioVersion.h>\n')
     f.write('#include <podio/CollectionBase.h>\n')
     f.write('\n')
 
-    f.write('\ntemplate <typename T> struct PodioTypeMap {};')
+    f.write('\ntemplate <typename T> struct PodioTypeMap {')
+    f.write('\n#if podio_VERSION >= PODIO_VERSION(0, 17, 0)')
+    f.write('\n    using collection_t = typename T::collection_type;')
```

Comment:
Is this still needed? Do we still rely on `PodioTypeMap<T>::collection_t` anywhere after all other changes? Is that reliance in Jana2?

Reply:
Yes, in JANA2, for now.